core-bundle/src/Controller/ContentElement/AbstractContentElementController.php

```diff
@@ -36,6 +43,8 @@ public function __invoke(Request $request, ContentModel $model, string $section,
             $response = $template->getResponse();
         }
 
+        $this->addSharedMaxAgeToResponse($response, $model);
```

Comment:
I don't think this is related to the PR?

Reply:
Imo it is. If a fragment controller now decides not to be shown for a time frame it needs to sets the shared max age as well. Otherwise the element might be visible even though the stop time is reached.

Reply:
Here's what was discussed when we added the method: https://github.com/contao/contao/pull/164#issuecomment-438633583